lib/gui/components/update-notifier/services/update-notifier.js

```diff
@@ -60,7 +60,9 @@ module.exports = function($http, $q, ModalService, UPDATE_NOTIFIER_SLEEP_TIME, M
           // The error status equals -1 if the request couldn't
           // be made successfully, for example, because of a
           // timeout on an unstable network connection.
-          if (error.status === -1) {
+          const ERROR_CODE_UNSUCCESSFULL_REQUEST = -1;
```

Comment:
> IMHO this constant should either be moved to the top of this file, or to the constants.js that I saw used elsewhere. 

The reason I put it here is that this error code is only relevant in the context of the `etcherLatestVersion` function. I'm worried that moving it somewhere else (even if its the top of the file), will make it seem that it applies somewhere else.

> And the comment above should be changed to not refer directly to -1.

Good idea!

> And there's only one 'l' in 'unsuccessful' ;)

Ah, fixed!